languages/i18n/vp.json

```
@@ -4332,24 +4332,24 @@
 	"passwordpolicies-policies": "Policies",
 	"passwordpolicies-policy-display": "<span class=\"passwordpolicies-policy\">$1 <code>($2)</code></span>",
 	"passwordpolicies-policy-displaywithflags": "<span class=\"passwordpolicies-policy\">$1 <code>($2)</code></span> <span class=\"passwordpolicies-policy-flags\">($3)</span>",
-	"passwordpolicies-policy-minimalpasswordlength": "Password must be at least $1 {{PLURAL:$1|character|characters}} long",
-	"passwordpolicies-policy-minimumpasswordlengthtologin": "Password must be at least $1 {{PLURAL:$1|character|characters}} long to be able to login",
-	"passwordpolicies-policy-passwordcannotmatchusername": "Password cannot be the same as username",
-	"passwordpolicies-policy-passwordcannotbesubstringinusername": "Password cannot be a substring within the username",
-	"passwordpolicies-policy-passwordcannotmatchblacklist": "Password cannot match a specific list of default passwords",
-	"passwordpolicies-policy-passwordcannotmatchdefaults": "Password cannot match a specific list of default passwords",
-	"passwordpolicies-policy-maximalpasswordlength": "Password must be less than $1 {{PLURAL:$1|character|characters}} long",
+	"passwordpolicies-policy-minimalpasswordlength": "Taenakaku mus na lestehobit $1 {{PLURAL:$1|character|characters}} kiraen",
+	"passwordpolicies-policy-minimumpasswordlengthtologin": "Taenakaku mus na lestehobit $1 {{PLURAL:$1|character|characters}} kiraen de deki zetulla",
+	"passwordpolicies-policy-passwordcannotmatchusername": "Taenakaku lakinae sama na brukjinnamae",
+	"passwordpolicies-policy-passwordcannotbesubstringinusername": "Taenakaku lakinae deki feunnena inye brukjinnamae",
+	"passwordpolicies-policy-passwordcannotmatchblacklist": "Taenakaku lakinae deki feunnena inye ein tumam f szvandz taenakaku",
+	"passwordpolicies-policy-passwordcannotmatchdefaults": "Taenakaku lakinae deki feunnena inye ein tumam f szvandz taenakaku",
+	"passwordpolicies-policy-maximalpasswordlength": "Taenakaku lakinae pluhobit na $1 {{PLURAL:$1|character|characters}} kiraen",

```

```suggestion
	"passwordpolicies-policy-maximalpasswordlength": "Taenakaku lakinae pluhobit na $1 kiraen",

```
Sama gruunvas na ova.